surface_apps/surface_normals/driver.py

```diff
@@ -0,0 +1,83 @@
+# ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+#  Copyright (c) 2024-2025 Mira Geoscience Ltd.                                '
+#                                                                              '
+#  This file is part of surface-apps package.                                  '
+#                                                                              '
+#  surface-apps is distributed under the terms and conditions of the MIT License
+#  (see LICENSE file at the root of this source code package).                 '
+# ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+import logging
+import sys
+from pathlib import Path
+
+import numpy as np
+from geoapps_utils.driver.driver import BaseDriver
+from geoapps_utils.utils.transformations import (
+    compute_normals,
+)
+from geoh5py.groups.property_group_type import GroupTypeEnum
+from geoh5py.objects import Points
+from geoh5py.shared.utils import fetch_active_workspace
+
+from surface_apps.surface_normals.options import SurfaceNormalsOptions
+
+
+logger = logging.getLogger(__name__)
+
+
+class SurfaceNormalsDriver(BaseDriver):
```

Comment:
Consider just calling this `Driver`, so we can have a generic `__main__` down the road. It is already in the `surface_normals` module anyway.

Reply:
done.  copied start mechanism from grid-apps and adapted iso-surfaces app as well